server/src/lib.rs

```diff
@@ -311,9 +311,7 @@ fn create_genesis_block(
         parent_beacon_block_root: Some(B256::ZERO),
         parent_hash: B256::ZERO,
         receipts_root: EMPTY_ROOT_HASH,
-        state_root: B256::new(hex!(
-            "30b67e4b5ef34eacb9e083c07fd5578982c2cb4e0ee1dc0a14d72b99a28ed80e"
-        )),
+        state_root: state_root_unhashed(genesis_config.l2_contract_genesis.alloc.clone()),
```

Comment:
Wow, that's amazing 🚀
While you're at it, can you pull the default genesis config here?

Reply:
Done in https://github.com/UmiNetwork/op-move/pull/488/commits/4bcb135b94fa981efdc44c1aca012a5e153e144c